hs_core/management/commands/reorder_authors.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+
+"""
+Fix duplicate author "order" values
+
+Related to https://github.com/hydroshare/hydroshare/issues/4695
+"""
+
+from django.core.management.base import BaseCommand
+from hs_core.models import BaseResource
+
+
+class Command(BaseCommand):
+    help = "Fix duplicate author 'order' values"
+
+    def handle(self, *args, **options):
+        resources = BaseResource.objects.filter(raccess__published=False).only('object_id', 'short_id')
+        for res in resources:
```

Comment:
Any resource for which we fix the author ordering, we also need to set the bag dirty flag for that resource so that next time when the bag is downloaded the metadata xml file will have the updated author ordering.